recipes/zyre/2.0.0/test_package/test_package.cpp

```diff
@@ -0,0 +1,9 @@
+#include <iostream>
+#include <zyre.h>
+#include <zyre_library.h>
+
+int main(void)
+{
+  zyre_test(false);
```

Comment:
Nice idea about using the selftest.
Applying the same idea to czmq (and libzmq) might be a good idea as well!

Reply:
Somehow, the selftest is throwing this `double free or corruption (fasttop)` error. The code itself looks properly regarding freeing memory from my point of view. Do you have an idea, why it is failing or can we turn of the check?

Reply:
As a first step, we can add the selftest of czmq to its recipe.
Then we can have more confidence that the problem lies not with czmq.
I guess you cannot reproduce the error on your end?